compile/compile_test.go

```diff
@@ -1111,13 +1118,13 @@ func TestCompilerOptimizationL1(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			optimizedExp := ast.MustParseModule(`
+			optimizedExp := ast.MustParseModuleWithOpts(`
 			package test
 
 			default p = false
-			p { data.test.q = X; X }
-			q { input.x = 1 }
-		`)
+			p if { data.test.q = X; X }
+			q if { input.x = 1 }
+		`, ast.ParserOptions{RegoVersion: ast.RegoV1})
```

Comment:
The generated policy is v1-only?

Reply:
No, but the compiler will have resolved/stripped any imports in the optimized module, so we need to compare it with a parsed module that has no imports. Could just as well have been compared against a pure v0 module, but since we're moving towards v1 anyways, I opted for that.

Of course, I could also have set `WithRegoVersion(ast.RegoV1)` on the compiler, and not use the import. Maybe I should do that instead 🤔, and not mix apples and oranges ..

Reply:
That would make sense.